test/unit/xiopak.unit.cpp

```diff
@@ -71,3 +82,57 @@ TEST(xiopak, file_operations)
     EXPECT_FALSE(xfExist(file1));
 
 }
+
+TEST(xiopak, path_functions) {
+
+    // Set up
+#define S(x) #x
+#define S_(x) S(x)
+#define S__LINE__ S_(__LINE__)
```

Comment:
Is this being used?

Reply:
Yes, it is used in the line below to convert the definition that CMake passed.

Reply:
Maybe I'm not looking at the right version of this file? I don't see **S__LINE__** anywhere.

Reply:
Oh yes. S__LINE__ is not used and it has been removed.